Sources/StreamChatSwiftUI/ChatChannel/ChatChannelViewModel.swift

```diff
@@ -428,6 +428,7 @@ open class ChatChannelViewModel: ObservableObject, MessagesDataSource {
     
     public func onViewAppear() {
         setActive()
+        utils.channelControllerFactory.currentChannelController = channelController
```

Comment:
Not sure if this is the best. Typically `currentChannelController` is set in `makeChannelController` but for brand new groups, we never call `makeChannelController`.